src/common/components/issue-filing-button.tsx

```diff
@@ -38,7 +42,12 @@ export class IssueFilingButton extends React.Component<IssueFilingButtonProps, I
     }
 
     public render(): JSX.Element {
+        if (isEmpty(this.props.userConfigurationStoreData)) {
```

Comment:
I think this latest change is an improvement (in that I think it makes more sense for this component to own knowledge of how to handle the data == null state), but I still think I would prefer to see the component render in a disabled or less-functional state, rather than not rendering anything at all; I think rendering null her is both inconsistent with the other buttons and prone to making the UI jittery during initial load.

Reply:
Oh, I misunderstood your request. I'm hesitate to make UX changes here without PM/Desginer's input, I'd rather make changes to make sure that the user config store state are loaded (the same way how we make sure other store data that the client view is depend on are loaded).